dashboard/src/app/dashboard/[dashboardId]/layout.tsx

```diff
@@ -49,12 +49,12 @@ export default async function DashboardLayout({ children, params }: DashboardLay
   return (
     <DashboardProvider>
       <DictionaryProvider dictionary={dictionary} initialLanguage={language}>
-        <section>
+        <section className="flex flex-col h-screen">
           <BATopbar />
```

Comment:
The top-bar is missing the border somehow, this is how it looked like before:
<img width="388" height="140" alt="image" src="https://github.com/user-attachments/assets/e4d7b0c8-32b1-4563-9396-dbd79aaa6106" />

Whereas now it's missing and is replaced by some gradient:
<img width="409" height="124" alt="image" src="https://github.com/user-attachments/assets/2584e073-48ac-4162-88f3-a1d9e66faf0e" />